contracts/svm/programs/express_relay/README.md

```diff
@@ -0,0 +1,46 @@
+# Express Relay Program
+
+This subdir contains the Express Relay program and its SDK to help integrating programs check permissioning and perform testing.
+
+## Express Relay Design
+
+The design of Express Relay on Solana utilizes the fact that Solana allows for multiple instructions to different programs within the same transaction. As a result, one of the instructions in the Express Relay instruction will be the `SubmitBid` instruction of the Express Relay program, using a `permission` account that represents the specific position/vault whose permissioning is being auctioned by Express Relay. This `permission` pubkey could represent the address of a user's position (or the keccak hash of relevant identifying information for a particular position). This `SubmitBid` instruction will also handle validating and distributing to the relevant parties (protocol, relayer, Express Relay admin) the bid.
```

Comment:
> will also handle validating and distributing to the relevant parties (protocol, relayer, Express Relay admin) the bid.

will also handle validating and distributing the bid to the relevant parties (protocols, relayer).